packages/features/calendar-cache/api/cron.ts

```diff
@@ -82,14 +84,18 @@ const handleCalendarsToUnwatch = async () => {
           if (error instanceof Error) {
             errorMessage = error.message;
           }
-          await SelectedCalendarRepository.updateById(id, {
-            error: `Error unwatching calendar: ${errorMessage}`,
+          log.error(`Error unwatching calendar ${externalId}`, safeStringify(error));
```

Comment:
Log the error with full details. The table's error field stores a shorter message